packages/framework/presence/src/presence.ts

```diff
@@ -36,12 +36,24 @@ export type ClientSessionId = SessionId & { readonly ClientSessionId: "ClientSes
  * @alpha
  */
 export const SessionClientStatus = {
+	/**
+	 * The session client is connected to the delta stream.
+	 * Can send to and receive updates from other session clients.
+	 */
 	Connected: "Connected",
+
+	/**
+	 * The session client is not connected to the delta stream.
+	 * Cannot send to or receive updates from other session clients.
+	 */
 	Disconnected: "Disconnected",
 } as const;
 
 /**
- * Type for the connection status of the {@link ISessionClient}.
+ * Represents the connection status of an {@link ISessionClient}.
+ *
+ * This type can be either `'Connected'` or `'Disconnected'`, indicating whether
+ * the session client is currently able to send and receive updates.
```

Comment:
Is there somewhere where we describe what happens if you try to send updates while in the disconnected state? Is the caller expected to check the state before sending? Out of scope for this PR probably but we should answer those questions somewhere.

Reply:
I think the comment for SessionClientStatus.Disconnect helps here for describing what happens if you try to send updates while disconnected: 

> /**
	 * The session client is not connected to the Fluid service.
	 * Cannot send to or receive updates from other session clients.
	 */

I also think that the current way clients should check if they are connected would be to directly check the connection state of the container rather than checking their own SessionClientStatus. I think the distinct SessionClientStatus use case would probably be presence related, for example displaying the connection status of session clients in an application. 

To your last sentence, I agree that this should be documented somewhere at some point.

Reply:
We know the answer and can state it. Clients don't send updates. They make state changes and notification requests. When disconnected state changes are kept locally and communicated to others upon reconnect. Notifications are discarded (silently). That statement for state handling is a little simplified. Exact details of handling are dependent on the value manager used for each state. For example, with the Latest* value managers intermediary states are "skipped".

Container connection status is a problem for Presence because it is op-centric. Currently Presence status matches that but we need to fix that. Non-trivial work.